R/pkg/R/DataFrame.R

```diff
@@ -3444,7 +3445,8 @@ setMethod("as.data.frame",
 #' @note attach since 1.6.0
 setMethod("attach",
```

Comment:
Could we also have some test for this one?

Reply:
good idea; added